apps/passport-client/components/shared/AppContainer.tsx

```diff
@@ -21,17 +21,25 @@ export function AppContainer({
 
   const col = bg === "gray" ? "var(--bg-dark-gray)" : "var(--bg-dark-primary)";
   return (
-    <Background color={col}>
-      <Container>
-        {children}
-        {error && <ErrorPopup error={error} onClose={onClose} />}
-      </Container>
-    </Background>
+    <>
+      <GlobalBackground color={col} />
+      <Background>
+        <Container>
+          {children}
+          {error && <ErrorPopup error={error} onClose={onClose} />}
+        </Container>
+      </Background>
+    </>
   );
 }
 
-const Background = styled.div<{ color: string }>`
-  background-color: ${(p) => p.color};
+const GlobalBackground = createGlobalStyle<{ color: string }>`
+  html {
```

Comment:
nit: `body` feels more correct here (since `html` wraps the <head> and other metadata too)

Reply:
the reason I chose `html` is that we're styling the background via `html` in our `css` files